react_app/src/threeVisualizer.js

```diff
@@ -0,0 +1,230 @@
+import * as three from 'three';
+import React from 'react';
+import { TrackballControls } from 'three/examples/jsm/controls/TrackballControls.js';
+
+const IMAGE_SIZE = 64;
+
+class ImageParams {
+  constructor(size) {
+    this.width = size;
+    this.height = size;
+  }
+}
+
+class Atlas {
+  constructor(image, numImages, numRows, numColumns) {
+    this.image = image;
+    this.width = image.width * numColumns;
+    this.height = image.height * numRows;
+    this.numImages = numImages;
+  }
+}
+
+class Point {
+  constructor(x, y, z) {
```

Comment:
This constructor deserves some comments. What is the usecase of the random assignment?

Reply:
Added class comments to all classes

Reply:
Should be commented there in newest commit, weird it wasn't showing up for you

Reply:
My question is when would someone call this with the values not present? Why do you assign them with random values (for what purpose)?

Reply:
It was useful earlier in the process (before there was actual data) for debugging purposes

Reply:
I can remove if you'd prefer, but imo it's not doing damage being there

Reply:
Please remove it. It will make maintenance harder and it can potentially introduce error/bug to someone who misuses it.

Reply:
Removed!